boot.mount

```diff
@@ -0,0 +1,11 @@
+[Unit]
+ConditionKernelCommandLine=ostree
+
+[Mount]
+What=/sysroot/boot
+Where=/boot
+Type=none
+Options=bind
+
+[Install]
+WantedBy=local-fs.target
```

Comment:
We don't need to list all units on the `dh_systemd_enable` call after @wjt's ba6e6c98c5be32076ca82880db3f9321478db14c.